.gitignore

```diff
@@ -1,3 +1,4 @@
 /vendor
 .phpbrew
 .phpcs.cache
+/build
```

Comment:
I notice that the `vendor/bin/phpunit --exclude-group mayignore --coverage-clover build/logs/clover.xml` command will generate the `build/logs` directory automatically and this `build` directory should be ignored from the Git version control.